src/components/SideTray/style.css

```diff
@@ -0,0 +1,56 @@
+.sideTray {
+  width: var(--rvr-side-tray-width);
+  height: 100vh;
+  background-color: var(--rvr-white);
+  position: fixed;
+  top: 0;
+  right: 0;
+  transition: transform 0.3s cubic-bezier(0, 0.52, 0, 1);
+  overflow: auto;
+  z-index: var(--rvr-off-canvas-drawer-z-index);
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.hide {
+  transform: translate3d(400px, 0, 0);
+}
+
+.show {
+  transform: translate3d(0, 0, 0);
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  overflow: hidden;
+  height: 100%;
+}
+
+.content {
+  flex-grow: 1;
```

Comment:
Any time something resizes with flex, you need `min-width: 0` / `min-height: 0` (depending on the flow).